e2e-tests/playwright/e2e/custom-theme.spec.ts

```diff
@@ -21,33 +22,22 @@ test.describe("CustomTheme should be applied", () => {
   });
 
   // eslint-disable-next-line no-empty-pattern
-  test("Verify that theme light colors are applied and make screenshots", async ({}, testInfo: TestInfo) => {
-    await themeVerifier.setTheme("Light");
-    await themeVerifier.verifyHeaderGradient(
-      "none, linear-gradient(90deg, rgb(248, 248, 248), rgb(248, 248, 248))",
-    );
-    await themeVerifier.verifyBorderLeftColor("rgb(255, 95, 21)");
-    await themeVerifier.takeScreenshotAndAttach(
-      "screenshots/custom-theme-light-inspection.png",
-      testInfo,
-      "custom-theme-light-inspection",
-    );
-    await themeVerifier.verifyPrimaryColors("rgb(255, 95, 21)");
-  });
-
-  // eslint-disable-next-line no-empty-pattern
-  test("Verify that theme dark colors are applied and make screenshots", async ({}, testInfo: TestInfo) => {
-    await themeVerifier.setTheme("Dark");
-    await themeVerifier.verifyHeaderGradient(
-      "none, linear-gradient(90deg, rgb(0, 0, 208), rgb(255, 246, 140))",
-    );
-    await themeVerifier.verifyBorderLeftColor("rgb(244, 238, 169)");
-    await themeVerifier.takeScreenshotAndAttach(
-      "screenshots/custom-theme-dark-inspection.png",
-      testInfo,
-      "custom-theme-dark-inspection",
-    );
-    await themeVerifier.verifyPrimaryColors("#ab75cf");
+  test("Verify theme colors are applied and make screenshots", async ({}, testInfo: TestInfo) => {
```

Comment:
Sonar Cloud doesn't like this syntaz with an empty object: {}. I guess you can fix it
```suggestion
  test("Verify theme colors are applied and make screenshots", async (_browser: { browser }, testInfo: TestInfo) => {
```

Reply:
That is actually considered the preferred way.

When I originally done something similar I ended up with the following error:

```
First argument must use the object destructuring pattern: _args

   at e2e/custom-theme.spec.ts:24

  22 |   });
  23 |
> 24 |   test("Verify theme colors are applied and make screenshots", async (_args, testInfo: TestInfo) => {
     |   ^
  25 |     const themes = ThemeConstants.getThemes();
  26 |
  27 |     for (const theme of themes) {
```

I guess we will just have to ignore sonarcloud for this one.

Reply:
Then you can use comment: // NOSONAR